tenacity/_asyncio.py

```diff
@@ -58,7 +58,7 @@ async def __anext__(self):
         while True:
             do = self.iter(retry_state=self._retry_state)
             if do is None:
-                raise StopAsyncIteration
+                raise StopAsyncIteration()
```

Comment:
unrelated?

Reply:
Formally bug, but yes: unrelated. It should be separate change